src/inject/utils/dom.ts

```diff
@@ -199,12 +199,32 @@ export function removeDOMReadyListener(listener: () => void) {
     readyStateListeners.delete(listener);
 }
 
+// `interactive` can and will be fired when their are still stylesheets loading.
+// We use certain actions that can cause a forced layout change, which is bad.
+export function isCompleteDomReady() {
```

Comment:
Something is wrong with the name. Is ready state "Complete" (true)? Or is DOM ready (true, but DOM is ready when state is "interactive" too)?

Reply:
I really should take a variable naming course.